lib/body.js

```diff
@@ -21,36 +21,36 @@ class Body {
 
     // Get the skills parameters from the parameter server to extract the
     // possible motions.
-    const param = new Param({
+    const parameter = new Param({
       ros,
       name: 'skills'
     });
 
     // Extract all relevant parameter from the parameter group
     const that = this;
-    param.get(param => {
+    parameter.get(parameter_ => {
```

Comment:
`parameter_` is a really strange name, what's wrong with using the same name?

Reply:
XO wants it updated